src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="DoubleSlider" :style="cssVars">
+    <div id="left-progressBar"></div>
+    <div id="right-progressBar"></div>
+    <input id="faderLeft" type="range" v-model="faderLeftValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderLeftValue">
+    <input id="faderRight" type="range" v-model="faderRightValue"
+      v-bind:min="sliderMin"
+      v-bind:max="sliderMax"
+      v-bind:step="sliderStep"
+      v-bind:aria-valuemin="sliderMin"
+      v-bind:aria-valuemax="sliderMax"
+      v-bind:aria-valuenow="faderRightValue">
+  </div>
+</template>
+
+<script>
+  import { library } from '@fortawesome/fontawesome-svg-core'
+  import { faKeyboard } from '@fortawesome/free-solid-svg-icons'
+  import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+  import Colored from '@/mixins/colored'
+  import Indicating from '@/mixins/indicating'
+  import Joined from '@/mixins/joined'
+  import Rounded from '@/mixins/rounded'
+  import Scaled from '@/mixins/scaled'
+  import Simplified from '@/mixins/simplified'
+
+  import Invertible from '@/mixins/invertible'
+  import Unactionable from '@/mixins/unactionable'
+
+  library.add(faKeyboard)
+
+  /**
+   * ### The double range slider is a input field very much like the regular range slider.
+   * the big diffrence is a double range slider allows users
+   * to select both a min and max.
+   * exp. The range slider can be useful for allowing users to select
+   * a specific price range when browsing products.
+   */
+  export default {
+    name: 'DoubleSlider',
+    data: function () {
```

Comment:
The order of options in the component definition are given [here](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Vocabulary/Code%20style).